cloudslang-enforcer/src/main/java/io/cloudslang/lang/enforcer/SpringCleansingRule.java

```diff
@@ -0,0 +1,198 @@
+/*******************************************************************************
+ * (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.lang.enforcer;
+
+import org.apache.maven.enforcer.rule.api.EnforcerRule;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleException;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleHelper;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.component.configurator.expression.ExpressionEvaluationException;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.lang.String.format;
+import static java.lang.String.valueOf;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.nio.file.Files.isRegularFile;
+import static java.nio.file.LinkOption.NOFOLLOW_LINKS;
+import static org.apache.commons.io.FileUtils.iterateFiles;
+import static org.apache.commons.io.FileUtils.readFileToString;
+import static org.apache.commons.lang3.StringUtils.countMatches;
+
+
+public class SpringCleansingRule implements EnforcerRule {
+
+    private static final java.lang.String JAVA = "java";
+    private static final String OPTIONAL_SPACER = "(\\s)*";
+    private static final String REGEX_DOT = "\\.";
+    private static final String OPTIONAL_STATIC = "(static )?";
+
+    private static final String ORG = "org";
+    private static final String SPRINGFRAMEWORK = "springframework";
+
+    private static final String SPACER_DOT_SPACER = OPTIONAL_SPACER + REGEX_DOT + OPTIONAL_SPACER;
+    private static final String regexImport = "import " + OPTIONAL_STATIC + OPTIONAL_SPACER + ORG +
+            SPACER_DOT_SPACER + SPRINGFRAMEWORK + "[^;]+" + ";" ;
+
+    private static final Pattern patternImport = Pattern.compile(regexImport);
+    private static final String regexCodeLine = OPTIONAL_SPACER + ORG + OPTIONAL_SPACER + REGEX_DOT +
+            OPTIONAL_SPACER + SPRINGFRAMEWORK + "(.|\\s)+;" ;
+
+    private static final Pattern patternCodeLine = Pattern.compile(regexCodeLine);
```

Comment:
a bit dangerous since cannot use `org.springframework.` at all (no comments, variable values right?) - however I don;t see a better approach by pure regex analysis - 👍